src/container.h

```diff
@@ -92,4 +92,53 @@ class RingDeque {
   T buffer_[CAPACITY];
 };
 
+//Stupid reimplementation of the deque
```

Comment:
I guess the only reason I didn't like it was because you can't append for many times as the capacity of the queue but maybe this is a standard behavior. 
So if I have to mirror the PRU queue that is made of 16 slots I have to make a deque with CAPACITY=17 so I made it work also with this deque.